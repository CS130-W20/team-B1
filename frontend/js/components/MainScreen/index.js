import React from 'react';
// import 'bootstrap/dist/css/bootstrap.css';
// import '../css/MainScreen.css';
import { Button } from 'reactstrap';


const MainScreen = () => {
  return (
  	<div className="MainScreen">
    	<div className="buttons">
	    	<Button className="button" color="success">Create Party
	    		<small> You are the host </small>
	    	</Button>
		    <Button className="button" color="danger">Join Party
		    	<small> Are you near a host? </small>
		    </Button>
  		</div>
  	</div>

  );
}

export default MainScreen;
