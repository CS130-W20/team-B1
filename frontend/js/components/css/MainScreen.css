body {
    margin: 0;
    height: 100%;
}

.MainScreen {
	/*border: 5px dotted red;*/
	position: relative;
	width: 100vw;
	height: 100vh;
}

.buttons {
	/*border: 5px dotted blue;*/
	position:absolute;
	bottom: 0;
	width: 100%;
	padding-left: 15px;
	padding-right: 15px;
}

.btn {
	margin-bottom: 10px;
	padding-bottom: 10px;
	border-radius: 15px;
	display: block;
	width: 100%;
	text-align: left;
	font-size: 30px;
}

/*green color button */
.btn-success {
  color: #fff;
  background-color: #65DDA5;
  border-color: #65DDA5;
}

/*red color button */
.btn-danger {
  color: #fff;
  background-color: #F16170;
  border-color: #F16170;
}


.btn small {
	display: block;
	font-size: 15px;
}



/*Join Party stuff*/
.JoinPartyList {
	height:100%;
	clear: both;
}

.title {
	padding-top: 10px;
	padding-bottom: 30px;
	padding-left: 15px;
	font-size: 25px;
	font-style: bold;
}

.subtitle {
	padding-top: 7px;
	padding-bottom: 15px;
	padding-left: 15px;
	font-size: 15px;
	font-style: bold;
	color: rgb(71, 70, 70);
}

.error {
	padding-left: 15px;
	color: darkred;
}

.list-group {
	max-height: 500px;
	margin: 0 5px;
    overflow:scroll;
    -webkit-overflow-scrolling: touch;	
}

.listCard {
	padding: 20px 5px;
	border-color: gray;
	border-radius: 10px;
	background-color: #EBF0F7;
	padding-bottom: 25px;
	margin-bottom: 15px;
}


.note {
	 flex-grow: 1;
	position:absolute;
	bottom: 0;
	width: 100%;
	text-align: center;
	margin-bottom: 10px;
	
}

.note p {
	margin-bottom: 0px;
}

/*Join Party 2*/
.enterName {
	padding-left: 15px;
	padding-right: 15px;
}

.enterName input {
	width: 100%;
	border-radius: 10px;
	padding: 20px 2px;
}

.enterCode {
	padding-left: 15px;
	padding-right: 15px;
}

.enterCode input {
	width: 100%;
	border-radius: 10px;
	padding: 20px 2px;
}


/* Party Joined stuff */
.listQueue {
	padding: 20px 5px;
	border-color: red;
	border-radius: 10px;
	background-color: #EBF0F7;
	margin-bottom: 15px;
	overflow: break-word;
}


.input-group.md-form.form-sm.form-1 input {
  border: 1px solid #bdbdbd;
  border-top-right-radius: 0.25rem;
  border-bottom-right-radius: 0.25rem;
}

/*search bar*/
.mr-sm-2{
	border-radius: 10px;
}

.searchDiv {
	padding: 20px 5px;
}

.buttonCard {
	align-items: right;

}

.SkipButton{
	font-size: 20px;
	text-align: center;
}

.AddButton {
	font-size: 20px;
	text-align: center;
}

.albumArtwork {
	width:80px;
	height:80px;
}

.column {
  float: left;
}

.left, .right {
  width: 25%;
}

.middle {
  width: 50%;
}


