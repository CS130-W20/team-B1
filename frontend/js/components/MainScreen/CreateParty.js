import React from 'react';
import 'bootstrap/dist/css/bootstrap.css';
import '../css/MainScreen.css';
import { Button } from 'reactstrap';


const CreateParty = () => {
  return (
    <div className="CreateParty">
    	<div className="buttons">
      	<Button className="button" color="success"> Login to Spotify
      		<small> You are the life of the party! </small>
      	</Button>
    	</div>
    </div>

  );
}

export default CreateParty;
