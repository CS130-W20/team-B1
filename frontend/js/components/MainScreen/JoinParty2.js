import React from 'react';
import 'bootstrap/dist/css/bootstrap.css';
import '../css/MainScreen.css';
import { Button } from 'reactstrap';


const JoinParty2 = () => {
  return (
  	<div className="JoinParty2">
  	    <div class="title">
	    	<h1>What is your name? </h1>
  		</div>

  		<div class="enterName">
	    	<input type="text"  value= "Name"/>
  		</div>

    	<div className="buttons">
	    	<Button className="button" color="success">Login to Spotify
	    		<small> Lets get rockin' </small>
	    	</Button>
  		</div>
  	</div>

  );
}

export default JoinParty2;
