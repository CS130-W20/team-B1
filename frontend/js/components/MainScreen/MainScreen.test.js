import React from 'react';
import renderer from 'react-test-renderer';

import MainScreen from './index';

describe ('Testing Spotify Login Page', () => {
    it('correctly renders', () => {
        const wrapper = renderer.create(<MainScreen />).toJSON();
        expect(wrapper).toMatchSnapshot();
    });
});