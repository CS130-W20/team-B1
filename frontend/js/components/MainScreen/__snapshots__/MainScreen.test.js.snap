// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Spotify Login Page correctly renders 1`] = `
<div
  className="MainScreen"
>
  <div
    className="buttons"
  >
    <button
      aria-label={null}
      className="button btn btn-success"
      onClick={[Function]}
    >
      Create Party
      <small>
         You are the host 
      </small>
    </button>
    <button
      aria-label={null}
      className="button btn btn-danger"
      onClick={[Function]}
    >
      Join Party
      <small>
         Are you near a host? 
      </small>
    </button>
  </div>
</div>
`;
